routers/web/user/profile.go

```diff
@@ -90,6 +91,28 @@ func Profile(ctx *context.Context) {
 		ctx.Data["RenderedDescription"] = content
 	}
 
+	// Fetches user's .profile/README.md and adds it to the users profile (if it exists)
+	resp, err := http.Get(setting.AppURL + ctx.ContextUser.Name + "/.profile/raw/README.md")
```

Comment:
Why not just read the file through git? I think there are setups which can't make http requests to the own host using the app url.